### PR TITLE
Add Baion878 PCB

### PR DESCRIPTION
**Pull request by ojthetiny**
## Description

Add support for Baion878 TKL PCB line

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/baion878/hsa/config.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Ensure we jump to bootloader if the RESET keycode was pressed */
```
Comment (waffle87) [inline keyboards/baion878/hsa/config.h:20]:
```suggestion
// Copyright 2024 ojthetiny
// SPDX-License-Identifier: GPL-2.0-or-later

#pragma once

// Ensure we jump to bootloader if the QK_BOOT keycode was pressed
```

File: keyboards/baion878/hsa/config.h
Comment (waffle87) [inline keyboards/baion878/hsa/config.h:1]: This file should be moved to the top keyboard level, as it's the same across all revisions.

File: keyboards/baion878/hsa/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+	[3] = LAYOUT(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+	),
+
```
Comment (waffle87) [inline keyboards/baion878/hsa/keymaps/default/keymap.c:42]:
```suggestion
		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT, KC_TRNS,  						 KC_UP,
		KC_LCTL, KC_LGUI, KC_LALT,								 KC_SPC,  							KC_RALT, 	KC_RGUI, 	KC_RCTL, 				KC_LEFT, KC_DOWN, KC_RGHT
	)
```

File: keyboards/baion878/hsa/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+	[3] = LAYOUT(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+	),
+
```
Comment (waffle87) [inline keyboards/baion878/hsa/keymaps/via/keymap.c:42]:
```suggestion
		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT, KC_TRNS,  						 KC_UP,
		KC_LCTL, KC_LGUI, KC_LALT,								 KC_SPC,  							KC_RALT, 	KC_RGUI, 	KC_RCTL, 				KC_LEFT, KC_DOWN, KC_RGHT
	)
```

File: keyboards/baion878/hsi/config.h
Comment (waffle87) [inline keyboards/baion878/hsi/config.h:1]: This file should be removed, as per above comment.

File: keyboards/baion878/hsi/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+	[3] = LAYOUT(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+	),
+
```
Comment (waffle87) [inline keyboards/baion878/hsi/keymaps/default/keymap.c:42]:
```suggestion
		KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT, KC_TRNS,   						 KC_UP,
		KC_LCTL, KC_LGUI, KC_LALT, 									KC_SPC,  								KC_RALT, KC_RGUI, KC_RCTL, 						KC_LEFT, KC_DOWN, KC_RGHT
	)
```

File: keyboards/baion878/hsi/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+	[3] = LAYOUT(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+	),
+
```
Comment (waffle87) [inline keyboards/baion878/hsi/keymaps/via/keymap.c:42]:
```suggestion
		KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT, KC_TRNS,   						 KC_UP,
		KC_LCTL, KC_LGUI, KC_LALT, 									KC_SPC,  								KC_RALT, KC_RGUI, KC_RCTL, 						KC_LEFT, KC_DOWN, KC_RGHT
	)
```

File: keyboards/baion878/readme.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+* Hardware Availability: Pre-Order
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make baion878/sl:default
+
+Flashing example for this keyboard:
+
+    make baion878/sl:default:flash
```
Comment (waffle87) [inline keyboards/baion878/readme.md:16]:
```suggestion
    make baion878/hsa:default
    make baion878/hsi:default
    make baion878/sl:default

Flashing example for this keyboard:

    make baion878/hsa:default:flash
    make baion878/hsi:default:flash
    make baion878/sl:default:flash
```

File: keyboards/baion878/sl/config.h
Comment (waffle87) [inline keyboards/baion878/sl/config.h:1]: This file should be removed, as per above comment.

File: keyboards/baion878/sl/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+	[3] = LAYOUT(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+	),
+
```
Comment (waffle87) [inline keyboards/baion878/sl/keymaps/default/keymap.c:42]:
```suggestion
		KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT, KC_TRNS,   				 KC_UP,
		KC_LCTL, KC_LGUI, KC_LALT, 							KC_SPC, 								KC_APP,  KC_RALT, KC_RGUI, KC_RCTL, 			KC_LEFT, KC_DOWN, KC_RGHT
	)
```

File: keyboards/baion878/sl/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+	[3] = LAYOUT(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+	),
+
```
Comment (waffle87) [inline keyboards/baion878/sl/keymaps/via/keymap.c:42]:
```suggestion
		KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT, KC_TRNS,   				 KC_UP,
		KC_LCTL, KC_LGUI, KC_LALT, 							KC_SPC, 								KC_APP,  KC_RALT, KC_RGUI, KC_RCTL, 			KC_LEFT, KC_DOWN, KC_RGHT
	)
```

File: keyboards/baion878/hsa/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/baion878/hsa/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2024 ojthetiny
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/baion878/hsa/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/baion878/hsa/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2024 ojthetiny
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/baion878/hsi/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/baion878/hsi/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2024 ojthetiny
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/baion878/hsi/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/baion878/hsi/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2024 ojthetiny
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/baion878/sl/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/baion878/sl/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2024 ojthetiny
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/baion878/sl/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/baion878/sl/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2024 ojthetiny
// SPDX-License-Identifier: GPL-2.0-or-later
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

File: keyboards/baion878/sl/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 ojthetiny
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/baion878/sl/keymaps/default/keymap.c:8]:
```suggestion
	[0] = LAYOUT_all(
```

File: keyboards/baion878/sl/keymaps/via/keymap.c [hunk 16]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 ojthetiny
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/baion878/sl/keymaps/via/keymap.c:8]:
```suggestion
	[0] = LAYOUT_all(
```

File: keyboards/baion878/sl/keymaps/via/rules.mk [hunk 17]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```
Comment (dunk2k) [inline keyboards/baion878/sl/keymaps/via/rules.mk:1]:
```suggestion
VIA_ENABLE = yes

```

File: keyboards/baion878/sl/rules.mk [hunk 18]
```diff
@@ -0,0 +1 @@
+# This file intentionally left blank
```
Comment (dunk2k) [inline keyboards/baion878/sl/rules.mk:1]:
```suggestion
# This file intentionally left blank

```

**Review by dunk2k — CHANGES_REQUESTED** [review]
1. add Community Layout support
    1. rename LAYOUT to LAYOUT_all to conform to PR guidelines
2. add EoF where missing

File: keyboards/baion878/sl/info.json
Comment (fauxpark) [inline keyboards/baion878/sl/info.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/baion878/hsi/info.json
Comment (fauxpark) [inline keyboards/baion878/hsi/info.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/baion878/hsa/info.json
Comment (fauxpark) [inline keyboards/baion878/hsa/info.json:1]: Please run `qmk format-json -i` on this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/baion878/config.h
Comment (zvecr) [inline keyboards/baion878/config.h:1]: This file should be removed, as it only duplicates default settings

```
platforms/chibios/boards/GENERIC_STM32_F072XB/configs/config.h
19:#    define EARLY_INIT_PERFORM_BOOTLOADER_JUMP TRUE
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/baion878/config.h
Comment (ojthetiny) [reply keyboards/baion878/config.h:1]: Removed

File: keyboards/baion878/sl/info.json
Comment (ojthetiny) [reply keyboards/baion878/sl/info.json:1]: done

File: keyboards/baion878/sl/info.json [hunk 19]
```diff
@@ -0,0 +1,127 @@
...
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "B10",
+        "on_state": 1,
```
Comment (fauxpark) [inline keyboards/baion878/sl/info.json:15]:
```suggestion
```

File: keyboards/baion878/readme.md [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+
+* Keyboard Maintainer: [Liv](https://github.com/ojthetiny)
+* Hardware Supported: Baion878 PCB
+* Hardware Availability: Pre-Order
+
+Baion878 Comes in 3 layouts:
+    
+    878HSA for ANSI Hotswap
+    878HSI for ISO Hotwsap
+    878SL for Universal Solderable
```
Comment (fauxpark) [inline keyboards/baion878/readme.md:13]:
```suggestion
 * 878HSA for ANSI Hotswap
 * 878HSI for ISO Hotwsap
 * 878SL for Universal Solderable
```

File: keyboards/baion878/hsa/info.json [hunk 21]
```diff
@@ -0,0 +1,123 @@
...
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "B10",
+        "on_state": 1,
```
Comment (fauxpark) [inline keyboards/baion878/hsa/info.json:15]:
```suggestion
```

File: keyboards/baion878/hsi/info.json [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "B10",
+        "on_state": 1,
```
Comment (fauxpark) [inline keyboards/baion878/hsi/info.json:15]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by fauxpark** [comment]
There is no need to continually merge master. Please do not do that.

File: keyboards/baion878/hsa/keymaps/default/keymap.c
Comment (drashna) [inline keyboards/baion878/hsa/keymaps/default/keymap.c:1]: Please replace the tabs with spaces.

File: keyboards/baion878/hsa/keymaps/via/keymap.c
Comment (drashna) [inline keyboards/baion878/hsa/keymaps/via/keymap.c:1]: Same

**Pull request closed without merging**